test/traversal/forEachDescriptor.js

```diff
@@ -1,7 +1,8 @@
 import interopRequireDefault from '../../src/helpers/interopRequireDefault';
 import forEachDescriptor from '../../src/traversal/forEachDescriptor';
 
-let mockComponents, mockVariations;
+let mockComponents; let
```

Comment:
This is a little weird

Reply:
oh wow, there are definitely some broken autofixers at play here 😔
will update manually.